$OpenBSD: patch-autogen_sh,v 1.2 2011/12/12 18:18:15 jasper Exp $

- don't force gmake being around, it's not used anyway.
- don't let this script run configure for us.

--- autogen.sh.orig	Mon Dec  6 01:29:02 2010
+++ autogen.sh	Mon Dec 12 10:03:03 2011
@@ -1,18 +1,6 @@
 #!/bin/sh
 
 
-MAKE=`which gnumake 2>/dev/null`
-if test ! -x "$MAKE" ; then MAKE=`which gmake` ; fi
-if test ! -x "$MAKE" ; then MAKE=`which make` ; fi
-HAVE_GNU_MAKE=`$MAKE --version|grep -c "Free Software Foundation"`
-
-if test "$HAVE_GNU_MAKE" != "1"; then 
-  echo Could not find GNU make on this system, can not proceed with build.
-  exit 1
-else
-  echo Found GNU Make at $MAKE ... good.
-fi
-
 if test ! -x "`which aclocal`"
 then
   echo "You need aclocal and autoconf to generate configure and Makefile."
@@ -40,12 +28,8 @@ $LIBTOOLIZE --dry-run --install > /dev/null 2>&1 && {
     LIBTOOLIZE_ARGS="--force --copy"
 }
 
-echo This script runs configure ...
-
 $LIBTOOLIZE $LIBTOOLIZE_ARGS
 
 which acinclude >/dev/null 2>&1 && acinclude
 which aclocal >/dev/null 2>&1 && aclocal
 autoconf
-
-./configure ${CFGARGS}  $*
